packages/@magic-sdk/provider/src/modules/wallet.ts

```diff
@@ -60,8 +60,8 @@ export class WalletModule extends BaseModule {
   }
 
   /* Prompt Magic's Wallet UI (not available for users logged in with third party wallets) */
-  public showUI() {
-    return this.request<boolean>(createJsonRpcRequestPayload(MagicPayloadMethod.ShowUI));
+  public showUI(onramperParams?: { [key: string]: string }) {
```

Comment:
We shall create a config object to handle all configurations data

Reply:
done